---
title: "Projects"
date: 2018-11-03T23:38:48+01:00
draft: false
---

# Data and digital skills

* **Languages:** Python, SQL, R, Java, Kotlin.
* **Python modules:** SciPy (NumPy, Pandas, Matplotlib)
* **Data:** SQLite, Microsoft Excel, Tableau, TAIR Database.
* **IDE:** Jupyter, PyCharm, Visual Studio Code, Android Studio.

# Laboratory skills

Strong background in project planning and experimental design and project
management experience.

### Molecular Biology and Genetics

DNA and RNA isolation, gene cloning, site-directed mutagenesis using GatewayÂ®
methodology and restriction enzymes cloning, PCR and DNA sequence analysis.
The study were performed in eukaryotic and prokaryotic cells (E.coli,
A.tumefaciens, S.cereviciae).
Gene disruption and genetic screenings and gene expression analysis in plants.

Transfection, selection and cloning of eukaryotic plant cell lines
(techniques learned in
[Max Planck Institute for Molecular Plant Physiology](http://www.mpimp-golm.mpg.de/2168/en))
and method competence in cell line development and cell cultures.
Characterisation of high expression clones.

### Biochemistry

Western blotting in native and no-native conditions, protein isolation,
Protein-protein interactions assays: (BiFC in tobacco plants and Arabidopsis,
Y2H (Yeast two Hybrid), Immunoprecipitation of complexes for identification
by Mass Spectrometry (LC-MS/MS).
Preparation and analysis of samples in genomics, transcriptomics and proteomics.

### Data Analysis
* **Phylogenetic data analysis:** FastME 2.0.
* **In vitro cloning analysis:** Genevestigator, Vector NTI 10, Serial cloner.
* **LC-MS/MS data analysis:** Q Exactive Benchtop Orbitrap LC-MS/MS.
* **Big data analysis in biology:** EMBL-EBI.
* **Protein and DNA sequence:** BLAST, MultiAlign.
